components/src/status_im/ui/components/react.cljs

```diff
@@ -61,7 +61,7 @@
 
 (def text-class ((get-class "Text")))
 (def text-input-class (get-class "TextInput"))
-(def image-class (get-class "Image"))
+(def fast-image-class (get-class "fast-image"))
```

Comment:
`get-class` retrieves class from react native package, but you need to get it from react-native-fast-image,  see how it works for image picker for example https://github.com/status-im/status-react/blob/f3aa376d167ed7d7dd5073b83f641cba7f861b03/components/src/status_im/ui/components/react.cljs#L202-L213